src/software/ai/hl/stp/play/corner_kick_play_test.py

```diff
@@ -112,9 +129,25 @@ def test_corner_kick_play_top_right(simulated_test_runner):
             }
         ],
         inv_always_validation_sequence_set=[[]],
-        inv_eventually_validation_sequence_set=[[]],
+        inv_eventually_validation_sequence_set=[
+            [
+                # Ball should always be in the field
+                BallEventuallyEntersRegion(
+                    regions=[tbots_cpp.Field.createSSLDivisionBField().fieldLines()]
+                ),
+                KickerAlwaysNotDoubleTouch(),
+            ]
+        ],
         ag_always_validation_sequence_set=[[]],
-        ag_eventually_validation_sequence_set=[[]],
+        ag_eventually_validation_sequence_set=[
+            [
+                # Corner kick should result in a successful pass to a teammate
+                # TODO: validate that kicker robot does not double touch the ball
```

Comment:
why is this a todo?

Reply:
pinging this. If we want to keep this TODO, we should add a issue number to it so it's tracked.